test/test_basic.py

```diff
@@ -1,15 +1,10 @@
 import unittest
 import os
 
-# skip the tests on GitHub actions
-if os.environ.get('CONDA_BUILD', 0) == '1':
-    skip_as_conda_build = True
-else:
-    skip_as_conda_build = False
+# skip test of UI imports on GitHub actions
+if os.environ.get('CONDA_BUILD', '0') == '0':
```

Comment:
Probably this is OK, but to be clear `CONDA_BUILD` is only present and set to `'1'` during a conda build. 

In all other occasions it may not be configured or misconfigured. That is why I had the test on the only thing we are sure: `CONDA_BUILD='1'` during a conda build.

Reply:
resolved by https://github.com/paskino/qt-elements/pull/42/commits/4773a5a957be014f84b72104a11da92b661b1123